village/src/components/Smurfs.js

```diff
@@ -1,10 +1,20 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
 import Smurf from './Smurf';
 
 class Smurfs extends Component {
-  // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
+  constructor() {
+    super();
+    this.state = {
+      smurfs: []
+    }
+  }
 
```

Comment:
If you turn this into a functional component, you're one step closer to fixing the rendering issue you were having after the axios post request.  Try keeping your state in App.js for things like your array of smurfs, and just pass the array down as props or something to this component.